### PR TITLE
Feature/344/add tags

### PR DESCRIPTION
**Pull request by rim23ma**
### Describe The Problem Being Solved:
1.  User can add a new tag in database.
2.  New tag becomes selected automatically.

### Checklist For Submitter

* [ ] I have documented all new functionality
* [x] I have screenshotted new UI changes and attached them to this PR

### CODE REVIEW

File: frontend/src/components/SearchTag/SearchTag.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+
+const tags = [
+  { label: 'Breakfast', value: 1 },
+  { label: 'Lunch', value: 2 },
+  { label: 'Dinner', value: 3 },
+  { label: 'Snaks', value: 4 },
+];
+
+class SearchTag extends Component {
```
[the remark this answers is not in the document]
Comment (vhoof) [reply frontend/src/components/SearchTag/SearchTag.js]: Come to think of it, what is this component for?

File: package.json [hunk 2]
```diff
@@ -11,5 +11,8 @@
     "lint": "cd frontend && npm run lint",
     "postinstall": "npm run build && npm run deploy",
     "start": "cd frontend && npm start"
+  },
+  "dependencies": {
+    "react-select": "^3.0.8"
```
Comment (vhoof) [inline package.json]: All dependencies go in the `frontend/package.json` file, not this one. You can remove these with `git checkout package.json && rm package-lock.json`

File: frontend/src/components/SearchTag/SearchTag.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    this.state = {
+      selectedOption: null,
+    };
+
+    this.handleChange = this.handleChange.bind(this);
+  }
+
+
+    handleChange = (selectedOption) => {
+      this.setState({ selectedOption });
```
[the remark this answers is not in the document]
Comment (rim23ma) [reply frontend/src/components/SearchTag/SearchTag.js]: handleChange = selectedOption => this.setState({ selectedOption });

File: frontend/src/components/SearchTag/SearchTag.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (rim23ma) [reply frontend/src/components/SearchTag/SearchTag.js]: I deleted SearchTag component, looks like we don't use it

File: .env. [hunk 4]
```diff
@@ -0,0 +1,5 @@
+SECRET_KEY_BASE=usuallysomereallylongstringofrandomcharactersbutitdoesntmatterinalocalenvironment
```
[the remark this answers is not in the document]
Comment (vhoof) [reply .env.]: Sorry, I think you may need to explicitly remove this file. `git rm .env.` should do it (you'll have to commit, of course).

File: package.json (hunk 2 above)
Comment (vhoof) [reply package.json]: This appears to still be here. You can just remove the changes (basically this whole dependencies block).

**Pull request closed without merging**